src/components/dialogmote/DialogmoteStedInfo.tsx

```diff
@@ -0,0 +1,15 @@
+import { BodyShort } from "@navikt/ds-react";
+import React from "react";
+import { DialogmoteDTO } from "@/data/dialogmote/types/dialogmoteTypes";
+
+const texts = {
+  moteSted: "Sted",
+};
+
+interface DialomoteStedInfoProps {
+  dialogmote: DialogmoteDTO;
+}
```

Comment:
Ikke veldig viktig, men har tenkt at vi egentlig ikke trenger å spesifisere hva `Props` gjelder for sånn generelt siden filen ofte impliserer hva det er snakk om 😊 Hadde kanskje vært mer gjennkjennbart om vi bare kalte det `Props` der det gir mening🤔 Hva tenker du? 

```suggestion
interface Props {
  dialogmote: DialogmoteDTO;
}
```

Reply:
Jo, enig i det. I hvert fall når de ikke eksporteres (som stort sett er tilfelle)  - kan fikse!